src/libPOW/pow.cpp

```diff
@@ -193,15 +193,14 @@ ethash_mining_result_t POW::MineGetWork(uint64_t blockNum,
   PoWWorkPackage work = {headerStr, seed, boundary, blockNum, difficulty};
 
   GetWorkServer::GetInstance().StartMining(work);
-  // -1: wait for the 1st accept result
-  auto result = GetWorkServer::GetInstance().GetResult(-1);
+  auto result = GetWorkServer::GetInstance().GetResult(timeWindow);
```

Comment:
the getwork server will sleep 'timeWindow' and return the last result, no chance for DS pow.

Reply:
we should change here
https://github.com/Zilliqa/Zilliqa/blob/4c192f3d61b7bd480aab0356ce29f0cb0d9639c2/src/libServer/GetWorkServer.cpp#L132-L136

Reply:
Yes, it is changed already. The node need to finish shard pow first then start to do DS pow. If somebody want to directly do DS pow, then he need to change the code himself

Reply:
https://github.com/Zilliqa/Zilliqa/blob/4c192f3d61b7bd480aab0356ce29f0cb0d9639c2/src/libServer/GetWorkServer.cpp#L118-L124

remove this sleep and use wait_ms in m_cvGotResult.wait_for?

Reply:
Yes, already modified like that